images/node/Dockerfile

```diff
@@ -23,6 +23,9 @@ RUN INSTALL_PKGS="libmnl libnetfilter_conntrack conntrack-tools openvswitch \
     yum clean all && \
     mkdir -p /usr/lib/systemd/system/origin-node.service.d /usr/lib/systemd/system/docker.service.d
 
+# Copy the cni plugins to the host file system when they are present, so that they can be shared with cri-o
+RUN if test -e /opt/cni/bin; then mkdir -p /exports/hostfs/opt/cni/bin/ && cp -r /opt/cni/bin/* /exports/hostfs/opt/cni/bin/; fi
```

Comment:
yes correct.  That wouldn't work in the Docker container case as these files won't be handled at installation time.